apps/baseline/models.py

```diff
@@ -279,19 +290,38 @@ class WorksheetName(models.TextChoices):
         verbose_name=_("Livelihood Zone Baseline"),
     )
     worksheet_name = models.CharField(max_length=20, choices=WorksheetName.choices, verbose_name=_("Worksheet name"))
-    cell_range = models.CharField(max_length=10, verbose_name=_("Cell range"))
-    previous_value = models.CharField(max_length=255, verbose_name=_("Previous value before correction"))
-    value = models.CharField(max_length=255, verbose_name=_("Corrected value"))
+    cell_range = models.CharField(max_length=20, verbose_name=_("Cell range"))
+    previous_value = models.JSONField(verbose_name=_("Previous value before correction"))
```

Comment:
`previous_value` and `value` also need `blank=True` so that we can capture an empty string to match/replace.